gems/pending/spec/util/miq-password_spec.rb

```diff
@@ -234,32 +234,26 @@
     end
 
     it "clears key caches" do
-      MiqPassword.add_legacy_key("v0_key", :v0)
-      MiqPassword.add_legacy_key("v1_key")
-      MiqPassword.v2_key
+      v0 = MiqPassword.add_legacy_key("v0_key", :v0)
+      v1 = MiqPassword.add_legacy_key("v1_key")
+      v2 = MiqPassword.v2_key
 
-      expect(MiqPassword.v0_key).to_not be_nil
-      expect(MiqPassword.v1_key).to_not be_nil
-      expect(MiqPassword.v2_key).to_not be_false
+      expect(MiqPassword.all_keys).to match_array([v2, v1, v0])
 
       MiqPassword.key_root = nil
 
-      expect(MiqPassword.v0_key).to be_nil
-      expect(MiqPassword.v1_key).to be_nil
-      expect(MiqPassword.v2_key).to be_false
+      expect(MiqPassword.all_keys).to be_empty
     end
 
     it "clears key caches" do
```

Comment:
Did we really have the same "it" string twice?  That seems bad.  Perhaps rename this one to say "clears legacy key caches"?